external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -5,7 +5,7 @@
     <!-- Given that xunit packages bring with them part of the framework, we need to specify a runtime in order to get the assets
          This RID value doesn't really matter, since the assets we are copying are not RID specific, so defaulting to Windows here
     -->
-    <NugetRuntimeIdentifier>win7-x64</NugetRuntimeIdentifier>
+    <NugetRuntimeIdentifier>win7-$(ArchGroup)</NugetRuntimeIdentifier>
```

Comment:
If the RID does matter then we should also update the comment. Why isn't sni coming in from the runtime.depproj? https://github.com/dotnet/corefx/blob/master/external/runtime/runtime.depproj#L36 that is where all the other native dependencies come from.